packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -120,10 +120,10 @@ const buttonSizeToSpinnerSizeMap: Record<
   NonNullable<BaseButtonProps['size']>,
   SpinnerProps['size']
 > = {
-  xsmall: 'medium',
+  xsmall: 'small',
   small: 'medium',
-  medium: 'large',
-  large: 'xlarge',
+  medium: 'medium',
+  large: 'large',
```

Comment:
Why are these changing 🤔

Reply:
Earlier we made up these sizes as placeholders until spinner was finalized on design. These are the new correct size mappings